mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -0,0 +1,479 @@
+#!/usr/bin/env python3
+
+
+import csv
+import getopt
+import os
+import subprocess
+import sys
+import math
+from prettytable import PrettyTable
+from prettytable import from_csv 
+from datetime import date
+
+# global variables.
+mlirBuildDir = './bin'
+mlirMIOpenDriver = 'mlir-miopen-driver'
+mlirROCmRunner = 'mlir-rocm-runner'
+rocprof = '/opt/rocm/bin/rocprof'
+MIOpenDriver = os.path.expanduser('~/MIOpen/build/bin/MIOpenDriver')
```

Comment:
I'd either have this be a relative path from the script directory or an argument. I'm not sure it's guaranteed that $WORKSPACE = $HOME in Jenkins, and the MIOpen build shouldn't be happening in `~/MIOpen/` - this has a decent chance of conflicting with other builds that'll happen in parallel

Reply:
This happens in a docker container. There is no other builds in parallel.

Reply:
It still feels like bad style, as far as Jenkins goes, to go outside the workspace

Reply:
Confining ourselves to the workspace means that, no matter what weird environment we decide to move to in the future, cleanWs() will still do a cleanup